src/Eval.ml

```diff
@@ -111,13 +128,35 @@ let extern_map =
     "dbl_printInt",   int_fun (fun n -> print_int n; v_unit);
     "dbl_readLine",   unit_fun (fun () -> VStr (read_line ()));
     "dbl_exit",       int_fun exit;
+    "dbl_negInt64",      int64_unop Int64.neg;
+    "dbl_addInt64",      int64_binop Int64.add;
+    "dbl_subInt64",      int64_binop Int64.sub;
+    "dbl_mulInt64",      int64_binop Int64.mul;
+    "dbl_divInt64",      int64_binop Int64.div;
+    "dbl_modInt64",      int64_binop Int64.rem;
+    "dbl_andInt64",      int64_binop Int64.logand;
+    "dbl_orInt64",       int64_binop Int64.logor;
+    "dbl_xorInt64",      int64_binop Int64.logxor;
+    "dbl_lslInt64",      int64_int_op Int64.shift_left;
+    "dbl_lsrInt64",      int64_int_op Int64.shift_right_logical;
+    "dbl_asrInt64",      int64_int_op Int64.shift_right;
+    "dbl_eqInt64",       int64_cmpop ( = );
+    "dbl_neqInt64",      int64_cmpop ( <> );
+    "dbl_gtInt64",       int64_cmpop ( > );
+    "dbl_ltInt64",       int64_cmpop ( < );
+    "dbl_geInt64",       int64_cmpop ( >= );
+    "dbl_leInt64",       int64_cmpop ( <= );
+    "dbl_intToInt64",    int_fun (fun n -> VNum64 (Int64.of_int n));
+    "dbl_int64ToInt",    int64_fun (fun n -> VNum (Int64.to_int n));
+    "dbl_int64ToString", int64_fun (fun n -> VStr (Int64.to_string n));
```

Comment:
This list is getting longer and longer. Maybe we should consider moving it to separate file.